projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.ts

```diff
@@ -100,4 +96,26 @@ export class PropertyViewComponent implements OnInit, OnDestroy {
         this.addButtonIsVisible = true;
         this.propID = undefined;
     }
+
+    /**
+     * Given a resource property, check if an add button should be displayed under the property values
+     *
+     * @param prop the resource property
+     */
+    addValueIsAllowed(prop: PropertyInfoValues): boolean {
+        // temporary until CkEditor is integrated
```

Comment:
Did we already create a DSP task for that?

Reply:
I don't believe so

Reply:
I found this in my drafts. I guess I forgot to click 'create' 😅 
https://dasch.myjetbrains.com/youtrack/issue/DSP-682